app/controllers/api/v1/classifications_controller.rb

```diff
@@ -34,8 +34,16 @@ def project
     index
   end
 
+  def project_offset
```

Comment:
the naming here is bugging me...i think the reason is that this is really just a modified project index route with some extra params.